e2e/pages/RegistrationDetails/RegistrationDetailsPage.ts

```diff
@@ -76,6 +79,13 @@ class RegistrationDetails {
     this.tileInformationStatus = this.page.getByTestId(
       'registration-activity-detail-status',
     );
+    (this.tileDetailsDropdownIcon = this.page.getByTestId(
+      'registration-notification-dropdown-icon',
+    )),
+      (this.preferredLanguageDropdown = this.page.getByTestId(
+        'preferred-language-dropdown',
+      ));
+    this.updateReasonTextArea = this.page.locator('textarea');
```

Comment:
Can you add a test id here?

Reply:
I made a change tho given how our FE is constructed it is a bit of workaround. Tell me what you think.